src/ucp/core/ucp_ep.c

```diff
@@ -292,26 +300,21 @@ ucs_status_t ucp_ep_init_create_wireup(ucp_ep_h ep,
     ucp_ep_config_key_reset(&key);
     ucp_ep_config_key_set_params(&key, params);
 
-    /* all operations will use the first lane, which is a stub endpoint */
     key.num_lanes             = 1;
-    key.lanes[0].rsc_index    = UCP_NULL_RESOURCE;
-    key.lanes[0].dst_md_index = UCP_NULL_RESOURCE;
-    key.am_lane               = 0;
-    key.wireup_lane           = 0;
-    key.tag_lane              = UCP_NULL_LANE; /* should be initialized only if
-                                                  HW TM is supported */
-    key.am_bw_lanes[0]        = 0;
-    key.rma_lanes[0]          = 0;
-    key.rma_bw_lanes[0]       = 0;
-    key.amo_lanes[0]          = 0;
-
+    if (ucp_worker_sockaddr_is_cm_proto(ep->worker)) {
+        key.cm_lane           = 0;
+    } else {
+        /* all operations will use the first lane, which is a stub endpoint */
+        key.wireup_lane       = 0;
+        key.am_lane           = 0;
```

Comment:
`dst_md_index` of wireup lane is `UCP_MAX_MDS` (initialized in reset) now. Is it ok?

Reply:
As I mentioned above, this should work because both (64 and 255) are not valid values for MD index